svc/GridExportService.ts

```diff
@@ -61,6 +62,9 @@ export class GridExportService extends HoistService {
         throwIf(!(isFunction(columns) || isArray(columns) || ['ALL', 'VISIBLE'].includes(columns)),
             'Invalid columns config - must be "ALL", "VISIBLE", an array of colIds, or a function'
         );
+        throwIf(!isBoolean(track),
```

Comment:
not sure we need this runtime checking especially with typescript....  

I don't think users will find this confusing, and the risk is fairly low if they do!